bindings/java/mac_run.me

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+cd `dirname $0`
+
+#unset _JAVA_OPTIONS
+if [ "x$JAVA_HOME" = "x" ]; then
+    JAVA_HOME=`/usr/libexec/java_home`
```

Comment:
Instead of making it macos-specific, I've chosen to check for `/usr/libexec/java_home`'s existence.